www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+
+The `[...nextauth].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](usage/next-auth) for more information on NextAuth.js, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
+
+<sub>(with NextAuth.js)</sub>
+
+## `src/pages/api/trpc`
+
+The `trpc` folder contains the tRPC API entry point.
+
+<sub>(with tRPC)</sub>
+
+### `src/pages/api/trpc/[trpc].ts`
+
+The `[trpc].ts` file is the tRPC API entry point. It is used to handle tRPC requests. See [tRPC usage](usage/trpc#-pagesapitrpctrpcts) for more information on this file, and [Next.js Dynamic Routes Docs](https://nextjs.org/docs/routing/dynamic-routes) for info on catch-all/slug routes.
+
+<sub>(with tRPC)</sub>
+
+## `src/server`
+
+The `server` folder is used to clearly separate server-side code from client-side code.
+
+<sub>(with tRPC and/or Prisma)</sub>
+
+## `src/server/common`
+
+The `common` folder contains commonly re-used server-side code.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/common/get-server-auth-session.ts`
+
+The `get-server-auth-session.ts` file is used to get the NextAuth.js session on the server-side. See [NextAuth.js usage](usage/next-auth#usage-with-trpc) for more information.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+## `src/server/db`
+
+The `db` folder contains the Prisma client.
+
+<sub>(with Prisma)</sub>
+
+### `src/server/db/client.ts`
+
+The `client.ts` file is used to instantiate the Prisma client at global scope. See [Prisma usage](usage/prisma#prisma-client) for more information.
+
+<sub>(with Prisma)</sub>
+
+## `src/server/trpc`
+
+The `trpc` folder contains the tRPC server-side code.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/context.ts`
+
+The `context.ts` file is used to create the context used in tRPC requests. See [tRPC usage](usage/trpc#-servertrpccontextts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/trpc.ts`
+
+The `trpc.ts` file is used to export procedure helpers. See [tRPC usage](usage/trpc#-servertrpctrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `src/server/trpc/router`
+
+The `router` folder contains the tRPC routers.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/_app.ts`
+
+The `_app.ts` file is used to merge tRPC routers and export them as a single router, as well as the type definitions. See [tRPC usage](usage/trpc#-servertrpcrouterts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/auth.ts`
+
+The `auth.ts` file is an example tRPC router utilizing the `protectedProcedure` helper to demonstrate how to protect a tRPC route with NextAuth.js.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/trpc/router/example.ts`
+
+The `example.ts` file is an example tRPC router utilizing the `publicProcedure` helper to demonstrate how to create a public tRPC route.
+
+<sub>(with tRPC)</sub>
```

Comment:
These are all covered in the tRPC docs so I think it can be squashed into one header with a quick summary